admin/services/data-access/administration-message.data.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const R = require('ramda')
+
+const sqlService = require('./sql.service')
+const TYPES = sqlService.TYPES
+const administrationMessageDataService = {}
+
+/**
+ * Create a new service message
+ * @param data
+ * @return {Promise.<*>}
+ */
+administrationMessageDataService.sqlCreate = async (data) => {
+  await sqlService.create('serviceMessage', data)
+}
+
+/**
+ * Updates the service message
+ * @param data
+ * @return {Promise.<*>}
+ */
+administrationMessageDataService.sqlUpdate = async (data) => {
+  const params = [
+    {
+      name: 'title',
+      value: data.title,
+      type: TYPES.NVarChar
+    },
+    {
+      name: 'message',
+      value: data.message,
+      type: TYPES.NVarChar
+    },
+    {
+      name: 'updatedByUser_id',
+      value: data['updatedByUser_id'],
+      type: TYPES.Int
+    }
+  ]
+  const sql = `
+    UPDATE [mtc_admin].serviceMessage
```

Comment:
This SQL update will update multiple rows as the PK is not included.  I suspect you think there will only ever be one row, but for that to be the case it must be part of the model at the SQL DDL level.

Reply:
@jon-shipley this method is removed.